kb/ostis_web_platform_components_specification.scs

```diff
@@ -0,0 +1,7 @@
+ostis_web_platform_repository
+	<- concept_repository;
+	-> rrel_components: ..components_addresses;;
+
+..components_addresses
+	-> ... (* -> rrel_address: [https://github.com/ostis-ai/ims.ostis.kb];; *);;
```

Comment:
Incorrect addresses specification model

Reply:
Actual version of specification is in #63.